ratis-grpc/src/main/java/org/apache/ratis/grpc/server/GrpcLogAppender.java

```diff
@@ -64,6 +66,7 @@
   private volatile StreamObserver<AppendEntriesRequestProto> appendLogRequestObserver;
 
   private final GrpcServerMetrics grpcServerMetrics;
+  private final StateMachine stateMachine;
```

Comment:
We may use getServer().getStateMachine().  Indeed StateMachine is not needed; see the other comment.

Reply:
Thanks for the review. Yes, will do.